File: Python_Programming/Day_5/program27.py
'''
    Description : Write a program that accprts frequency & prints
                  the below pattern : 
    Input : 4
    Output : 4 3 2 1
'''
def Display(No):

    for i in range(No,0 ,-1):
        print(i , "\t" , end = "")


def main():
    Value = 0

    print("Enter the frequency : ",end = "")
    Value = int(input())

    Display(Value)

if __name__ == "__main__":
    main()

File: C_Programming/Day_5/program27.c
/*
    Description : Write a program that accprts frequency & prints
                  the below pattern : 
    Input : 4
    Output : 4 3 2 1
*/

#include<stdio.h>

void Display(int iNo)
{
    int iCnt = 0;

    for(iCnt = iNo ; iCnt >= 1 ; iCnt--)
    {
        printf("%d\t",iCnt);
    }

}

int main()
{
    int iValue = 0;

    printf("Enter the frequency : ");
    scanf("%d",&iValue);

    Display(iValue);

}
